From f53bd0fc061702d205c7e15ca57c73e16f73c171 Mon Sep 17 00:00:00 2001
From: "Teng, Jin Chung" <jin.chung.teng@intel.com>
Date: Mon, 9 Jan 2023 09:58:38 +0800
Subject: [PATCH 1/2] vaapi: decoder: Adding back missing DMABuf in src caps

Found out dmabuf src caps is missing in vaapi decoder in 1.20.3 version.
https://gitlab.freedesktop.org/gstreamer/gstreamer/-/commit/f04dd7f33100ead30d98e5c76d42ee24290fa771

upstream status: merged

Signed-off-by: Lim Siew Hoon <siew.hoon.lim@intel.com>
Signed-off-by: Teng, Jin Chung <jin.chung.teng@intel.com>
---
 gst/vaapi/gstvaapidecode.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gst/vaapi/gstvaapidecode.c b/gst/vaapi/gstvaapidecode.c
index 1eb6208..aed1f99 100644
--- a/gst/vaapi/gstvaapidecode.c
+++ b/gst/vaapi/gstvaapidecode.c
@@ -71,6 +71,7 @@ char *gst_vaapidecode_sink_caps_str = NULL;
 
 static const char gst_vaapidecode_src_caps_str[] =
     GST_VAAPI_MAKE_SURFACE_CAPS "; "
+    GST_VIDEO_CAPS_MAKE_WITH_FEATURES(GST_CAPS_FEATURE_MEMORY_DMABUF, GST_VAAPI_FORMATS_ALL) " ;"
 #if (GST_VAAPI_USE_GLX || GST_VAAPI_USE_EGL)
     GST_VAAPI_MAKE_GLTEXUPLOAD_CAPS "; "
 #endif
-- 
2.25.1

